crates/rspack_util/src/itoa.rs

```diff
@@ -0,0 +1,8 @@
+pub use itoa::Buffer;
+
+#[macro_export]
+macro_rules! itoa {
```

Comment:
I prefer a function here (if possible) instead of a macro

Reply:
The function can cause `temporary value` issues, and using `to_owned` may result in some performance loss. I am not sure how to better implement it.

Reply:
I see